builds/android/app/src/main/java/org/easyrpg/player/InitActivity.java

```diff
@@ -95,36 +95,22 @@ public void prepareData() {
      * copied to internal memory and executed.
      */
     private void startGameStandalone() {
-        // TODO : Verify that the standalone mode isn't broken
         AssetManager assetManager = getAssets();
-        String dataDir = getApplication().getApplicationInfo().dataDir;
-
-        // Standalone mode: Copy game in game folder to data folder and launch it
-        if (AssetUtils.exists(assetManager, "game")) {
-            Log.i("EasyRPG", "Standalone mode : a \"game\" folder is present in asset folder");
-            standaloneMode = true;
+        String gameDir = "";
 
-            // Copy game in internal memory
-            if (!(new File(dataDir + "/game").exists())) {
-                AssetUtils.copyFolder(assetManager, "game", dataDir + "/game");
-            }
-        }
-
-        // Standalone mode: Unzip game.zip
+        // Standalone mode: Launch the game inside "game.zip" in the APK
         if (AssetUtils.fileExists(assetManager, "game.zip")) {
             Log.i("EasyRPG", "Standalone mode : a \"game.zip\" file is present inside the asset folder");
             standaloneMode = true;
-
-            // Unzip game to internal memory
-            if (!(new File(dataDir + "/game").exists())) {
-                AssetUtils.unzipFile(assetManager, "game.zip", dataDir + "/game");
-            }
+            gameDir = "apk://game.zip";
         }
 
         if (standaloneMode) {
-            // Launch the game
-            DocumentFile gameFolder = DocumentFile.fromFile(new File(dataDir + "/game"));
-            Game project = new Game(gameFolder);
+            // Launch the game: FIXME
+            String saveDir = getApplication().getApplicationInfo().dataDir + "/Save";
```

Comment:
To be tested for a follow up PR: To make the savegames available putting them in ``/sdcard/Android/data/[APP]`` (``getExternalFilesDirs``) would be better.

Then you can access them via ADB. At least for now... until Google fully restricts this m(